pages/EventsAdmin/EventEditor.jsx

```diff
@@ -269,6 +272,9 @@ class EventEditor extends React.Component {
                         value={ this.state.format }
                         onChange={ this.onChange.bind(this, 'format') } />
 
+                    <Checkbox name='lockDecks' label='Lock Decks' labelClass='col-sm-4' fieldClass='col-sm-offset-3 col-sm-8'
```

Comment:
"Prevent users from making changes to their decks for the duration of the event"

Reply:
changed it